admin/src/scenes/phase0/components/CniField.jsx

```diff
@@ -77,57 +76,64 @@ export function CniField({
           "mb-[15px]",
           "flex",
           "items-center",
-          "justify-between",
           "rounded-[7px]",
-          "p-[30px]",
+          "px-4",
+          "py-6",
           {
-            "bg-amber-50": isDatePassed,
+            "bg-indigo-50": isDatePassed,
             "border-l-8": isDatePassed,
-            "border-amber-500": isDatePassed,
+            "border-indigo-400": isDatePassed,
             "bg-gray-50": !isDatePassed,
           },
           className,
         )}
         onMouseEnter={() => setMouseIn(true)}
         onMouseLeave={() => setMouseIn(false)}>
-        <div className="shrink-0 flex">
-          <UploadedFileIcon />
-          <div className="text-sm leading-5 font-medium">
-            <div className="flex mt-3 ml-2">
+        <div className={classNames("flex", "w-full", { "justify-between": !isDatePassed })}>
+          <div className="w-2/10">
+            <UploadedFileIcon />
+          </div>
+          <div className={classNames("text-sm", "leading-5", "font-medium", "w-7/10", { flex: !isDatePassed, "items-center": !isDatePassed })}>
+            <div className="flex ml-2">
               <p>Pièce d'identité</p>
               {isDatePassed && (
                 <>
                   <HiOutlineInformationCircle size={20} className="ml-2 text-gray-900" data-tip data-for={"CNI-tooltip"} />
                   <ReactTooltip id={"CNI-tooltip"} type="light" place="top" effect="solid" className="custom-tooltip-radius rounded-md !opacity-100 !shadow-md">
                     <div className={"w-[275px] list-outside !px-1 !py-2 text-left text-xs font-normal text-gray-600 "}>
-                      {"La date de péremption de la CNI est dépassé, pensez à vérifier qu’il n’y ait pas d’erreur de saisie."}
+                      {"La date de péremption de la CNI est dépassée, pensez à vérifier qu’il n’y ait pas d’erreur de saisie (non bloquant pour la validation du dossier)."}
                     </div>
                   </ReactTooltip>
                 </>
               )}
             </div>
             {isDatePassed && (
-              <div className="flex ml-2 mt-2">
-                <HiExclamation size={20} className="text-amber-400 mr-2" />
-                <p className="text-amber-800">Date de validité dépassée</p>
+              <div className="flex">
+                <HiInformationCircle size={30} className="text-indigo-500 mr-2" />
+                <p className="text-indigo-700">
+                  Date de validité dépassée (non bloquant pour la validation du dossier).{" "}
+                  <a href=" https://support.snu.gouv.fr/base-de-connaissance/instruction-des-dossiers-par-les-referents" className="underline">
```

Comment:
```suggestion
                  <a href=" https://support.snu.gouv.fr/base-de-connaissance/instruction-des-dossiers-par-les-referents" className="underline" target="_blank">
```
dans un nouvel onglet ?